catroid/src/org/catrobat/catroid/content/bricks/WhenNfcBrick.java

```diff
@@ -80,6 +80,10 @@ public WhenNfcBrick(WhenNfcScript script) {
 		this.oldSelectedNfcTag = null;
 		this.nfcTag = script.getNfcTag();
 		this.whenNfcScript = script;
+
+		if (script != null && script.isCommentedOut()) {
+			setCommentedOut(true);
+		}
```

Comment:
instead of copying the comment out status in the clone() method of each and every Brick I only copied it at places where clone() was called (...) and only when copying the comment-out status made sense. This way I had to change less files. Still, I am a bit dissatisfied because if someone adds a new feature where brick-cloning is used has to remember to copy the value as well. On the other hand, if someone adds a new Brick, she also has to think of copying the boolean in the clone() method.
Unfortunately I can't think of a good solution using inheritance either because clone() doesn't modify any member variables, therefore copying via super.clone() doesn't work either.
What do you think, should I leave it as it is, should I copy the boolean in every brick?
That's one thing I've already been pondering about while developing the feature.